### PR TITLE
Gnw adjustments

### PR DESCRIPTION
**Pull request by SkewedAskew**
Contains notable throw changes and some other minor fixes and tweaks

Hitbox Comparisons: https://imgur.com/a/WTuojlY

### General:
- [+] Run Acceleration (multiplier): 0.06 -> 0.08

### Jab:
- [+] Damage: 3.0% -> 4.5%
- [=] BKB: 22/15 -> 22
- [=] KBG: 25/20 -> 25

### Jab Finisher:
- [+] Damage: 3.0% -> 4.5%
- [/] Hitlag Multiplier: 2.0x -> 1.5x

### Forward Tilt:
- Late
  - [+] Chair legs hitbox angled to match early hit
  - [+] BKB (ground/air): 35/30 -> 35
  - [+] KBG (ground/air): 120/135 -> 135

### Forward Smash:
- [$] Adjusted animation; Gnw now lowers the torch the frame the hitboxes clear
- Late
  - [-] Flame hitbox moved inward
  - [$] Adjusted sfx
  - [-] Damage: 14.0/18.0% -> 14.0%
  - [R] BKB: 42/46 -> 42
  - [R] KBG: 105/100 -> 105
  - [-] Hitbox Size (flame): 5.3u -> 4.3u
- [-] FAF: 39 -> 40

### Down Smash:
- [$] Adjusted animation; hitboxes now appear the frame the hammers hit the ground

### Forward Air:
- Initial
  - [-] Outer hitbox moved in and down 1.0u
  - [-] KBG: 100 -> 96
- Early
  - [-] KBG: 100 -> 96

### Fire:
- [$] Matched trampoline motion to Gnw's altered frame data

### Forward Throw:
- [+] BKB: 100 -> 70
- [-] KBG: 40 -> 50

### Back Throw:
- [+] BKB: 100 -> 70
- [-] KBG: 40 -> 50

### Up Throw:
- [-] Damage: 12.0% -> 8.0%
- [R] BKB: 95 -> 60
- [R] KBG: 54 -> 110

### CODE REVIEW

**Review by SuddyN — APPROVED** [review]
https://user-images.githubusercontent.com/33226440/221720306-cec29c14-dad9-4aba-83a2-c6e9442f2c94.mp4

File: fighters/gamewatch/src/status.rs [hunk 1]
```diff
@@ -18,7 +18,7 @@ mod landing_attack_air;
 
 pub fn install() {
     // smashline::install_agent_init_callbacks!(gamewatch_init);
-    landing_attack_air::install();
```
Comment (jobrien97) [inline fighters/gamewatch/src/status.rs:21]: Why is this not being installed?

Comment (SuddyN) [reply fighters/gamewatch/src/status.rs:21]: I'll let Askew answer but it looks like this was preventing edge slipoffs

Comment (SkewedAskew) [reply fighters/gamewatch/src/status.rs:21]: Does essentially nothing now that dair's edge cancel is fixed

Comment (jobrien97) [reply fighters/gamewatch/src/status.rs:21]: Ah, I thought that was what fixed it lol

**Pull request merged**